README.md

```diff
@@ -272,6 +272,19 @@ in the `build` directory from above and then following the other `cmake` command
 cmake -DBOND_ENABLE_GRPC=TRUE -G "Visual Studio 14 2015 Win64" ..
 ```
 
+Alternatively, you can build and install Bond using the [vcpkg](https://github.com/microsoft/vcpkg/) dependency manager:
+
+```bash
```

Comment:
Since these are instructions for Windows, I'd recommend using the "batch" language identifier.

Reply:
Updated, thanks!